src/GitTools.jl

```diff
@@ -159,9 +159,6 @@ function fetch(io::IO, repo::LibGit2.GitRepo, remoteurl=nothing; header=nothing,
         LibGit2.Callbacks()
     end
     fancyprint && start_progress(io, bar)
-    if credentials === nothing
```

Comment:
Shouldn't this be used instead of being removed?

Reply:
I was unsure which direction we want to go with this.
I went with removing it as that seemed more conservative to me, but I am open to using it.
I will add a commit to use it.